WORKSPACE

```diff
@@ -20,16 +20,32 @@ http_archive(
 # https://github.com/tensorflow/quantum
 http_archive(
     name = "org_tensorflow",
-    sha256 = "e82f3b94d863e223881678406faa5071b895e1ff928ba18578d2adbbc6b42a4c",
-    strip_prefix = "tensorflow-2.1.0",
+    sha256 = "e52cda3bae45f0ae0fccd4055e9fa29892b414f70e2df94df9a3a10319c75fff",
+    strip_prefix = "tensorflow-2.11.0",
     urls = [
-        "https://github.com/tensorflow/tensorflow/archive/v2.1.0.zip",
+        "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.11.0.zip",
     ],
 )
 
+load("@org_tensorflow//tensorflow:workspace3.bzl", "workspace")
```

Comment:
[No change needed] I distinctly recall trying to use this stack of TF workspaces in the past, but it didn't work. Any idea what makes it possible now?

Reply:
that was just TF-side error... it requires pre-installed dependency libraries (e.g. numpy, etc). so I added a line to install them like `python3 -m pip install -r requirements.txt` in the internal infra `kokoro_build.sh`.